modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetrics.macro

```diff
@@ -1894,5 +1894,12 @@ definition {
 			echo("timeout exceeded");
 		}
 	}
+	macro notViewNodeCircleIcon {
+		AssertElementNotPresent(locator1 = "WorkflowAllItems#NODE_CIRCLE_ICON");
+	}
+
+	macro viewNodeCircleIcon {
```

Comment:
Macro already merged, updated and renamed to `viewCurrentNodeCircleIcon`. Please change in your testcase